diff --git a/src/backend/pyproject.toml b/src/backend/pyproject.toml
index 2180b1c..b7008f9 100644
--- a/src/backend/pyproject.toml
+++ b/src/backend/pyproject.toml
@@ -17,12 +17,12 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.11",
 ]
 description = "A simple video and phone conferencing tool, powered by LiveKit"
 keywords = ["Django", "Contacts", "Templates", "RBAC"]
 license = "MIT"
-requires-python = ">=3.13"
+requires-python = ">=3.11"
 dependencies = [
     "boto3==1.42.68",
     "Brotli==1.2.0",
